components/bottombar/bottombar.js

```diff
@@ -0,0 +1,8 @@
+angular.module('app').directive('bottombar', [bottombarDirective]);
+
+function bottombarDirective() {
+	return {
+        restrict: 'E',
```

Comment:
It's unnecessary to define the restrict attribute for that directive according to [angularJS](https://docs.angularjs.org/guide/directive) documentation: `A directive can specify which of the 4 matching types it supports in the restrict property of the directive definition object. The default is EA.`

Reply:
Sure, default is EA, which match Elemtns and Attributes. Restrict to E only matches with elements, so the directive can only be accessed with:
```
<bottombar></bottombar>
```
and not
```
<div bottombar></div>
```

just becouse I think this might be confusing. If you think that's a mistake, we can just remove that property and let's go ;)